data/domain/tome.tsx

```diff
@@ -164,7 +162,7 @@ export class TomeLine {
     }
 
     getLineUnlockLevel = (): number => {
-        return (500 + (50 * TomeLineUnlockLevel[this.index] + (10 * Math.max(0, TomeLineUnlockLevel[this.index] - 30) + 10 * Math.max(0, TomeLineUnlockLevel[this.index] - 50))));
+        return (500 + (50 * this.displayOrder + (10 * Math.max(0, this.displayOrder - 30) + 10 * Math.max(0, this.displayOrder - 50))));
```

Comment:
oh I see now, the fix here is the usage of `indexOf` in the `displayOrder` instead of the actual value of the index in the array.

No wonder I couldn't get it to match up properly when doing my own math lol.

Reply:
Yeah, I don't know how I ended up doing it this way x)